src/iop/crop.c

```diff
@@ -1329,12 +1333,25 @@ void gui_post_expose(struct dt_iop_module_t *self, cairo_t *cr, int32_t width, i
     g_object_unref(layout);
   }
 
+  cairo_set_line_width(cr, DT_PIXEL_APPLY_DPI(2.0) / zoom_scale);
+  double alpha = CLAMP(1.0 - (g_get_monotonic_time() - g->focus_time) / 2e6f, 0.0, 1.0);
```

Comment:
nice trick :)

Reply:
It works well because we do a full redraw on each mouse move. If that were optimised, the dimming would not be as smooth anymore...